install-dev/data/db_structure.sql

```diff
@@ -2500,7 +2500,7 @@ CREATE TABLE `PREFIX_stock_available` (
     `id_shop`, 
     `id_shop_group`
   )
-) ENGINE=ENGINE_TYPE DEFAULT CHARSET=utf8 COLLATION;
+) ENGINE = ENGINE_TYPE DEFAULT CHARSET = utf8 COLLATION;
```

Comment:
why do you add space here when it's easier to export structure from mysqldump command and it doesn't add its :/

Reply:
I thought that spaces around operators was the right format in sql. (https://solutioncenter.apexsql.com/rules-of-sql-formatting-commas-and-spacing-in-t-sql/)

But I can change it back for all places, if you want?

Reply:
Fixed